tests/Agent/IntegrationTests/UnboundedIntegrationTests/NServiceBus/NsbAsyncCmdHandlerTests.cs

```diff
@@ -13,12 +13,12 @@
 
 namespace NewRelic.Agent.UnboundedIntegrationTests.NServiceBus
 {
-    public abstract class NsbAsyncCmdHandlerTests<TFixture> : NewRelicIntegrationTest<TFixture>
+    public abstract class NsbAsyncCmdHandlerTestsBase<TFixture> : NewRelicIntegrationTest<TFixture>
```

Comment:
Let the naming wars begin! :) For posterity, I was trying to reduce the length of the filenames/folders written by the learning transport, but this is not one of the typenames that ended up being written to disk.

I agree the way you have it in this PR is now consistent with the rest of our tests.

Reply:
Yeah, I was trying to keep things in line the existing name.  I agree that it is probably time for another re-org of the names.